### PR TITLE
Replace local simpla behavior with simpla-element-behavior

### PR DESCRIPTION
**Pull request by seaneking**

### CODE REVIEW

File: src/simpla-text.html [hunk 1]
```diff
@@ -59,7 +60,15 @@
...
       get behaviors() {
-        return [ editor, inline, simpla ];
+        return [ SimplaBehaviors.Element, editor, inline ];
+      }
+
+      created() {
+        this.setupSimpla({
+          type: 'Text',
+          dataProperties: [ 'value' ],
+          updateOn: 'input'
```
Comment (bedeoverend) [inline src/simpla-text.html]: This should be an array I'm pretty sure?

**Comment by bedeoverend** [comment]
There's the comment above, but there's also some regressions introduced even after that's patched. Both in e2e and unit / integration tests, but I think most of those are in the upstream behavior, need to address those so that this doesn't get any regressions first.

File: src/simpla-text.html (hunk 1 above)
Comment (seaneking) [inline src/simpla-text.html]: whoops yep

**Comment by seaneking** [comment]
See comments on behavior PR, simpla-text should either store its value in a property or not use this behavior.

**Pull request merged**